src/material/core/tokens/_token-utils.scss

```diff
@@ -0,0 +1,77 @@
+@use 'sass:map';
+@use '@material/theme/custom-properties' as mdc-custom-properties;
+@use '@material/theme/theme' as mdc-theme;
+@use '../mdc-helpers/mdc-helpers';
+@use '../theming/palette';
+
+// Placeholder color config that can be passed to token getter functions when generating token
+// slots.
+$placeholder-color-config: (
+    primary: palette.$red-palette,
+    accent: palette.$red-palette,
+    warn: palette.$red-palette,
```

Comment:
Should we wrap these palettes in `mat.define-palette` to match what a real color config looks like?

Reply:
Fixed